lib/active_admin/views/components/table_for.rb

```diff
@@ -115,10 +115,8 @@ def render_data(data, item)
       end
 
       def is_boolean?(data, item)
-        if item.respond_to? :has_attribute?
-          item.has_attribute?(data) &&
-            item.column_for_attribute(data) &&
-            item.column_for_attribute(data).type == :boolean
+        if item.class.respond_to? :columns_hash
+          column = item.class.columns_hash[data.to_s] and column.type == :boolean
```

Comment:
> Line is too long. [81/80]

@timoschilling I'd prefer the maximum to be 90 or 100 characters. Do you have a strong opinion either way?

> Use `&&` instead of `and`.

Heh, static code analysis can only understand so much.